### DISCUSSION TITLE
Schrittziele / Schrittaufgaben

### DISCUSSION THREAD

**Discussion by henrybrink** [SE-TINF22B2/G5-DuoGradus#4] [Anforderungen]
Die Anwendung soll verschiedene Schrittziele bereitstellen, an denen der Nutzer teilnehmen kann.
Ein Schrittziel ist eine Aktivität bei der folgende Eigenschaften vorgegeben sind:

- Anzahl der Schritte
- Zeit in der die Schritte erreicht werden müssen
- Punkte

Eine Aufgabe muss von einem Benutzer ausgewählt werden, um diese zu starten.

**Comment by henrybrink** [comment]
Erweiterung der Idee:

Neben Schritten gibt es noch weitere Aktivitätsarten, z.B. "Radfahren" oder "Joggen", bei denen jeweils noch weitere Parameter überprüfe.
Eine Beispiel-Aktivität wäre:

> Fahrrad-Challange: Sammele in 50 Minuten 3km mit dem Fahrrad und mindestens 10 Höhenmeter

**Reply by benedictweis** [reply]
Das würde ich als Optionale Anforderung aufnehmen.
